types/storybook-addon-intl/index.d.ts

```diff
@@ -0,0 +1,30 @@
+// Type definitions for storybook-addon-intl 2.3
+// Project: https://github.com/truffls/storybook-addon-intl#readme
+// Definitions by: 9renpoto <https://github.com/9renpoto>
+// Definitions: https://github.com/truffls/storybook-addon-intl
+
+import { RenderFunction, Renderable } from "@storybook/react";
+
+export function register(): undefined;
+
+export type WithIntlProps = {
```

Comment:
I have not much experience with TS but what do you think about using the definition `IntlProvider.Props` from [@types/react-intl](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-intl/index.d.ts#L186)?

Reply:
I can't see it. :/ Did you push it?